sql/2019/01_JavaScript/01_02a.sql

```diff
@@ -14,7 +14,7 @@ FROM (
       url,
       type,
       respSize,
-      NET.HOST(url) IN (SELECT requestDomain FROM `httparchive.almanac.third_parties`) AS is_third_party
+      NET.HOST(url) IN (SELECT domain FROM `httparchive.almanac.third_parties`) AS is_third_party
```

Comment:
FYI for reviewers: this is an intentional change because I modified the schema of the `third_parties` table.